docs/ansible/roles/debops.apt/defaults-detailed.rst

```diff
@@ -12,6 +12,69 @@ simple strings or lists, here you can find documentation and examples for them.
    :local:
    :depth: 1
 
+.. _apt__ref_conf:
+
+apt__conf
+---------
+
+This list, along with ``apt__group_conf`` and ``apt__host_conf`` and
+can be used to manage APT configuration files through Ansible inventory. Each
+entry is a YAML dictionary with by the `Ansible copy module`_. See its
```

Comment:
"YAML dictionary with by the" seems to be missing "keys and values the same as the ones used"